examples/cpp_sycl/makefile_lnx

```diff
@@ -78,27 +78,36 @@ ifndef TBBROOT
 endif
 TBB_PATH = "$(TBBROOT)/lib/$(_IA)/gcc4.8"
 
-EXT_LIB := -lpthread -ldl -lOpenCL
+EXT_LIB := -Bdynamic -lpthread -ldl -lOpenCL
+
+ifeq ($(RES_EXT),so)
+    DAAL_LIB_EXT := -Bdynamic
+else
+    DAAL_LIB_EXT := -Bstatic
+endif
 
 ifeq ($(threading),sequential)
-    DAAL_LIB_T := $(DAAL_PATH)/libdaal_sequential.$(RES_EXT)
+    DAAL_LIB_T := $(DAAL_LIB_EXT),-ldaal_sequential
 else
     override threading = parallel
-    DAAL_LIB_T := $(DAAL_PATH)/libdaal_thread.$(RES_EXT)
+    DAAL_LIB_T := $(DAAL_LIB_EXT),-ldaal_thread
     EXT_LIB += $(addprefix -L,$(TBB_PATH)) -ltbb -ltbbmalloc
 endif
 
-DAAL_LIB := $(DAAL_PATH)/libdaal_core.$(RES_EXT) $(DAAL_LIB_T)
+DAAL_LIB := -L$(DAAL_PATH),$(DAAL_LIB_EXT),-ldaal_core,-Bdynamic -Wl,$(DAAL_LIB_T)
+#LEVEL0_LIB := libze_intel_gpu.so.0.8 $l0_lib/libze_loader.so 
+LEVEL0_LIB := -L$(DRIVER_PATH) -lze_loader 
```

Comment:
DRIVER_PATH variable? Who will define it at customers level???

Reply:
fixed